spec/factories/allocations.rb

```diff
@@ -24,9 +28,9 @@
       Faker::Number.number(digits: 7)
     end
 
-    nomis_offender_id do
-      Faker::Alphanumeric.alpha(number: 10)
-    end
+    # nomis_offender_id do
+    #   Faker::Alphanumeric.alpha(number: 10)
+    # end
 
```

Comment:
This is the only feed back I have so far! Did you mean to leave this commented out?